source/Octopus.Tentacle.Contracts/Builders/StartScriptCommandV3AlphaBuilder.cs

```diff
@@ -11,12 +11,12 @@ public class StartScriptCommandV3AlphaBuilder
         readonly List<string> arguments = new();
         readonly Dictionary<ScriptType, string> additionalScripts = new();
         StringBuilder scriptBody = new(string.Empty);
-        ScriptIsolationLevel isolation = ScriptIsolationLevel.FullIsolation;
+        ScriptIsolationLevel isolation = ScriptIsolationLevel.NoIsolation;
         TimeSpan scriptIsolationMutexTimeout = TimeSpan.FromMilliseconds(int.MaxValue);
         string scriptIsolationMutexName = "RunningScript";
         string taskId = Guid.NewGuid().ToString();
         ScriptTicket scriptTicket = new UniqueScriptTicketBuilder().Build();
-        TimeSpan? durationStartScriptCanWaitForScriptToFinish = TimeSpan.FromSeconds(5);
+        TimeSpan? durationStartScriptCanWaitForScriptToFinish;
```

Comment:
These were the defaults in the `LatestStartScriptCommandBuilder`, so changed these defaults